secrets/resource_secret_scope.go

```diff
@@ -122,10 +118,6 @@ func kvDiffFunc(ctx context.Context, diff *schema.ResourceDiff, v any) error {
 	if len(kvLst) == 0 {
 		return nil
 	}
-	client := v.(*common.DatabricksClient)
```

Comment:
We can remove the whole `kvDiffFunc` function - it was necessary only to check if service principal is used or not